src/NuGet.Services.V3/Registration/Models/RegistrationPackageVulnerability.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json;
+using NuGet.Protocol.Catalog;
+
+namespace NuGet.Protocol.Registration
+{
+    public class RegistrationPackageVulnerability : PackageVulnerability
```

Comment:
I don't think `RegistrationPackageVulnerability` should extend `PackageVulnerability` given that we in the future we may add properties to `PackageVulnerability` that shouldn't be added to `RegistrationPackageVulnerability`.

For example, registration has its own copy of the catalog entry with `RegistrationCatalogEntry`. I think it'd be best to follow that model.

Reply:
Yes, agree. Thanks.